test/test_transformer.py

```diff
@@ -1190,6 +1190,13 @@ def test_transformer_authority_filter():
     assert transformer.description == "Ballpark geographic offset from WGS 84 to ETRS89"
 
 
+@pytest.mark.parametrize(
+    "allow_superseded", [(True,), (False,), (123456789,), ("blah",)]
+)
+def test_transformer_allow_superseded(allow_superseded):
+    Transformer.from_crs(6319, "EPSG:4326+5703", allow_superseded=allow_superseded)
```

Comment:
I would prefer to have a test where `.from_crs(..., allow_superseded=True)` has different behavior than `.from_crs(..., allow_superseded=False)`. But I could not find one. This test simply ensures that there is no issue using the kwarg.